frontend/packages/dev-console/src/components/GuidedTourTile.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import {
+  Card,
+  CardActions,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardHeaderMain,
+  Dropdown,
+  DropdownItem,
+  GalleryItem,
+  KebabToggle,
+  Title,
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import {
+  GuidedTourItem,
+  TourStatus,
+} from '@console/app/src/components/guided-tours/utils/guided-tour-typings';
+import { removeGuidedTours } from '../utils/guided-tours/guided-tour-action';
+import './GuidedTourTile.scss';
+
+type TourItem = GuidedTourItem & TourStatus;
+type GuidedTourTileProps = {
+  tours: TourItem[];
+};
+
+const GuidedTourTile: React.FC<GuidedTourTileProps & DispatchProps> = ({ tours, removeTours }) => {
```

Comment:
Using redux for this seems like overkill when there's only 1 spot in the app where we need this functionality.
Wouldn't something simple like this do the trick:
```
const [showTile, setShowTile] = React.useState(!localStorage.get(HIDE_TOUR_TILE_STORAGE_KEY));
const onRemove = () => {
  localStorage.set(HIDE_TOUR_TILE_STORAGE_KEY, true);
  setShowTile(false);
};
...
return showTile ? ... : null;
```

@rohitkrai03 thoughts?

Reply:
I agree with @christianvogt here, it doesn't make sense to use redux if the state is only ever going to be used in this component.